DataManagement/BranchTutorial.Rmd

```diff
@@ -85,87 +85,100 @@ knitr::include_graphics("Pics/6. add_url.png")
 <br/><br/>
 <br/><br/>
 
-## Branch
+## What is a branch?
 
 You have forked a repository and are ready to start working on the project. Let's talk about branches.
 
-A branch lets you develop code or test an idea without affecting the original project. Once you are happy with the new code, or the idea is tested and working you can merge the branch with master branch. If the idea did not work or you have another idea, the branch can be deleted. None of this will affect the original project.
+A branch lets you develop code or test an idea without affecting the original project. This means you can bravely develop code that works or does not work. Once you are happy with the new code, or the idea is tested and working you can merge the branch you are working on with master branch. If the idea did not work or you have another idea, the branch can be deleted and nobody ever needs to know about it. Make a new branch and start all over again. None of this will affect the original project.
```

Comment:
Top tip - write one sentence per line.
New sentence, new line. 

Should we talk about merging with the master branch here? On your own project this is really useful, but if you are working on a fork, you really shouldn't merge. Not sure how best to handle this.